tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -120,6 +120,8 @@ public TracerSettings(IConfigurationSource source)
                                    // default value
                                    false;
 
+            TracerStatsEnabled = source?.GetBool(ConfigurationKeys.TracerStatsEnabled) ?? false;
```

Comment:
Unless I'm missing something we shouldn't allow this feature to be turned on now, right? I mean don't we need to plug in normalization and obfuscation before?

Reply:
Well I don't think we should add code that can't be activated. Rather, we should wait for the normalization/obfuscation PRs before merging this one

Reply:
Works for me